src/DataSource/Types.hs

```diff
@@ -77,6 +77,8 @@ newtype App a = App { runAppBase :: ReaderT Config IO a }
              , MonadReader Config
              , MonadIO
              , MonadBase IO
+             , MonadCatch
```

Comment:
I really think we don't need this, unless you want to re-export, but that's a whole different story.

Reply:
Ah, sorry, I thought that was from the module exports :grin: 
Should have looked in more detail.